otx/algorithms/common/tasks/training_base.py

```diff
@@ -190,6 +190,11 @@ def project_path(self):
         """Return output path with logs."""
         return self._output_path
 
+    @property
+    def config(self):
```

Comment:
I think the name "config" is too broad given that this is only using for mm training backend. 
How about final_recipe or final_config?

Reply:
First of all, anomaly is also using an attribute called **config**. I've named it as a common attribute name, which seems to be up for discussion.